api/proto/banyandb/database/v1/schema.proto

```diff
@@ -156,8 +156,10 @@ message IndexRule {
   }
   // type is the IndexType of this IndexObject.
   Type type = 3 [(validate.rules).enum.defined_only = true];
+  // sorted indicates whether the index is for sorting.
+  bool sorted = 4;
```

Comment:
Adding the flag(4) would make incompatible change.

The good practice should be the largest index for a new field.

Reply:
You should move `sorted` to 7, and accept the value absent with a default value, which could keep the compatibility.